src/marks/linearRegression.d.ts

```diff
@@ -2,23 +2,85 @@ import type {ChannelValue, ChannelValueDenseBinSpec, ChannelValueSpec} from "../
 import type {Data, MarkOptions, RenderableMark} from "../mark.js";
 import type {BinOptions, BinReducer} from "../transforms/bin.js";
 
+/** Options for the linearRegressionX and linearRegressionY marks. */
 interface LinearRegressionOptions extends MarkOptions, BinOptions {
+  /**
+   * A channel for the dependent variable **x**.
+   */
   x?: ChannelValueSpec;
+  /**
+   * A channel for the dependent variable **y**.
+   */
   y?: ChannelValueSpec;
+  /**
+   * The series, for multiple regressions.
+   */
   z?: ChannelValue;
+  /**
+   * The confidence interval in [0, 1), or 0 to hide bands; defaults to 0.95.
+   */
   ci?: number;
+  /**
+   * The distance (in pixels) between samples of the confidence band; defaults to 4.
+   */
   precision?: number;
+  /**
+   * 🌶 How is this used?
+   */
   reduce?: BinReducer;
```

Comment:
It’s from maybeDenseInterval. It’s the same idea as line and area. For example for areaY you can specify an interval along _x_, and then reduce defaults to _first_ to extract the first value in _y_ for the current interval in _x_.

Reply:
I've documented what works. Can you open an issue with what doesn't? (It's not clear to me what's missing here.)